### PR TITLE
Added Pikaur (AUR tool) support

### PR DESCRIPTION
**Pull request by ewilly**

### CODE REVIEW

File: src/package.cpp [hunk 1]
```diff
@@ -341,6 +341,13 @@ QString Package::removeColorCodesFromStr(const QString &str)
   ret = ret.remove("[0m");
   ret = ret.remove("[1m");
   ret = ret.remove("\u001B");
+  ret = ret.remove("[030m");
+  ret = ret.remove("[031m");
+  ret = ret.remove("[032m");
+  ret = ret.remove("[033m");
+  ret = ret.remove("[034m");
+  ret = ret.remove("[035m");
+  ret = ret.remove("[036m");
```
Comment (actionless) [inline src/package.cpp]: you can just run `pikaur --color=never` instead

Comment (actionless) [reply src/package.cpp]: my bad, i see it's already used like that. is there any color output still left with that option?

Comment (ewilly) [reply src/package.cpp]: your right this comes from a first commit which is not anymore need by using  --color=never

**Comment by DrDoctor13** [comment]
Can a param be added for pikaur's `--devel` and `--needed` arguments?

**Comment by aarnt** [comment]
Hi @ewilly,
Did you test your changes with other AUR tools? I mean, no regressions?

**Comment by ewilly** [comment]
Hi @aarnt ,
I didn't see any regression

**Pull request merged**